app/main/views/briefs.py

```diff
@@ -116,8 +115,8 @@ def send_thank_you_email_to_responders(brief, brief_response, brief_response_url
     ess = brief.get('essentialRequirements', [])
     nth = brief.get('niceToHaveRequirements', [])
 
-    ess = zip(ess, brief_response.get('essentialRequirements', []))
-    nth = zip(nth, brief_response.get('niceToHaveRequirements', []))
+    ess = list(zip(ess, brief_response.get('essentialRequirements', [])))
+    nth = list(zip(nth, brief_response.get('niceToHaveRequirements', [])))
 
```

Comment:
Dont understand the current codes with this:
zip(ess, brief_response.get('essentialRequirements', []))

Same thing is inserted into zip twice.

Reply:
This is getting the essential requirements and the nice to have requirements from a seller's response.  We need two statements here.